backend/s3/s3.go

```diff
@@ -1390,7 +1415,19 @@ func s3Connection(opt *Options) (*s3.S3, *session.Session, error) {
 		SessionToken:    opt.SessionToken,
 	}
 
-	lowTimeoutClient := &http.Client{Timeout: 1 * time.Second} // low timeout to ec2 metadata service
+	defaultClient := getClient(opt)
+	defaultClient.Timeout = 1 * time.Second
```

Comment:
I think you've got the timeouts the wrong way round for `lowTimeoutClient` and `defaultClient`